README.md

```diff
@@ -1,7 +1,8 @@
 # KOBCC
 
 ## Abstract
-KochiOrgBook Community Cloud is a cloud interface for the city of Kochi. The Community Cloud project aims to develop a platform where participants can build, host and operate their own blockchain applications and related functions on the blockchain.
+KochiOrgBook Community Cloud is a provision given to an entity who does not have an infrastructure to build, host and run their nodes. Here, the entity can be an association like an association of lawyers, an association of doctors or it can be citizens who are trying to present their proof. It can be considered as a pool of infrastructure components on which their nodes can be implemented. The major stakeholders of the project include the citizens of Kochi, the associations who are to be onboarded and the various service providers.
+
 ## Dependent Projects
 
 The project is dependent on KOBman.
```

Comment:
@adithyak04 How did you come up with this?

Reply:
@asa1997 It was from the previous corrections made by Vinod Sir